Microsoft.Toolkit.Uwp.UI.Controls/MetadataControl/MetadataControl.cs

```diff
@@ -78,13 +87,22 @@ public string AccessibleSeparator
         }
 
         /// <summary>
-        /// Gets or sets he <see cref="MetadataUnit"/> to display in the control.
+        /// Gets or sets the <see cref="MetadataItem"/> to display in the control.
         /// If it implements <see cref="INotifyCollectionChanged"/>, the control will automatically update itself.
         /// </summary>
-        public IEnumerable<MetadataUnit> MetadataUnits
+        public IEnumerable<MetadataItem> Items
```

Comment:
I think normally we'd use `IList<MetadataItem>` vs. enumerable here? Though this isn't a standard ItemsControl type setup, so maybe it doesn't matter.

Reply:
I'm usually only using `IEnumerable<T>` or `IReadOnlyList<T>/IReadOnlyCollection<T>`  for such cases to be explicit on the fact that the control will not alter the list content.